Source/Core/UICommon/ResourcePack/ResourcePack.cpp

```diff
@@ -18,7 +18,7 @@
 
 namespace ResourcePack
 {
-constexpr char TEXTURE_PATH[] = "Load/Textures/";
+constexpr char TEXTURE_PATH[] = "Textures/";
```

Comment:
If we're following the mnemonic, this should be `HIRES_TEXTURES_DIR DIR_SEP`.

Reply:
Makes sense.  Maybe I'll just remove this variable then?

Reply:
Well it is used in several different places, so I dunno.

Reply:
Ah yeah, thought it was only used in two places, looks like three though.  I'll keep it

Reply:
Perhaps it would make more sense to drop it entirely here and move the `HIRES_TEXTURES_DIR` reference to ResourcePackManager.cpp? `path` only ever has that appended to it in `Install` and `Uninstall` anyways.

Reply:
I'm changing that in #8360 so I went with the first option

Reply:
Ah, okay makes sense to keep it like this then.